integration/pages/BaseTestPage.js

```diff
@@ -0,0 +1,68 @@
+const BasePage = require('./BasePage');
```

Comment:
Should this be RosterTestPage since this is for the Roster Test App?

Reply:
There is a `RosterTestPage.js` file, my next PR will include it. Since this PR is to setup Mocha and Selenium I didn't include that file. 

This `BasePage.js` encapsulates some common WebDriver operations, e.g. `find`, `findAll`, `isDisplayed`, ...
These are very basic operations, every tests will use them.

The `BaseTestPage.js` encapsulates opening demo URL, joining the meeting and leaving the meeting operations and corresponding checks. These are common operations that every tests will use too.

Their relation is:
`RosterTestPage extends BaseTestPage`
`BaseTestPage extends BasePage`

Reply:
Got it. Is there a reason we can't have everything in this PR?

Reply:
Just Vidya wants me to break down PRs and tasks. So I raised the PR based on the SIM tasks. 😃

Reply:
Yeah but each tests should be self-contain and is testable. I am fine with pushing the demo but this needs to have something to test on.

Reply:
Got it, I will address these comments first and push other commits here later.